README.md

```diff
@@ -1,15 +1,15 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
+# Quran online course 
```

Comment:
- [ ]  Please recorded a video presenting the features of the project.
- [ ] The maximum length of the video is 5 minutes.
- [ ] The video contains a clear and objective description about the project in professional English. 
- [ ] The topic and purpose of the project are made clear.
- [ ] The video contains a demo of the project, showing the pages in different screen sizes.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Thanks for review.

Reply:
- [ ] @NurkaAmre great presentation, however, you forgot to describe your project what is it about eg `offer a learning platform for Islamic studies` also you forgot to highlight the piece of code you found interesting or proud of making.